client/src/document/organisms/metadata/index.tsx

```diff
@@ -1,3 +1,6 @@
+import * as React from "react";
```

Comment:
Why did you add this?

Reply:
I do not think I explicitly added it, or perhaps I did just because we used to always include it. I know since React 17 you don't really need to do that anymore.